internal/nexodus/nexodus.go

```diff
@@ -673,6 +672,19 @@ func (ax *Nexodus) symmetricNatDisco() error {
 		isSymmetric = stunAddr.String() != stunAddr2.String()
 	}
 
+	if stunServer1 != "" {
```

Comment:
Is `stunServer1` right? It looks like you wanted `stunAddr` and `stunAddr2` ?

Reply:
😵‍💫 good catch, here is the output:

```
2023-05-10T22:24:26.481Z	DEBUG	nexodus/nexodus.go:676	first NAT discovery STUN request returned: 71.31.152.254:39802
2023-05-10T22:24:26.481Z	DEBUG	nexodus/nexodus.go:682	second NAT discovery STUN request returned: 71.31.152.254:39802
```